tests/unit/subscriptions/test-class-wc-payments-product-service.php

```diff
@@ -240,4 +240,66 @@ public function test_get_product_data_for_subscription() {
 		];
 		$this->assertEquals( $this->product_service->get_product_data_for_subscription( $subscription ), $expected_result );
 	}
+
+	/**
+	 * Test $product_service->get_tax_rates_for_item().
+	 */
+	public function test_get_tax_rates_for_item() {
+		$mock_order        = WC_Helper_Order::create_order();
+		$mock_subscription = new WC_Subscription();
+		$mock_subscription->get_parent( $mock_order );
+
+		$mock_items      = $mock_order->get_shipping_methods();
+		$mock_order_item = array_pop( $mock_items );
+
+		update_option( 'woocommerce_calc_taxes', 'no' );
+
+		$this->assertSame( [], $this->product_service->get_tax_rates_for_item( $mock_order_item, $mock_subscription ) );
```

Comment:
This is a nitpick so feel free to ignore it.

What do you think about separating this bit from the rest of this test. So we would have `test_get_tax_rates_for_items_with_no_tax_data` and `test_get_tax_rates_for_items_with_tax_data` or something like that?

Reply:
Thanks for the suggestion! I've separated these unit tests and updated existing ones in 5d19bf2